pkg/icingadb/delta.go

```diff
@@ -40,101 +38,83 @@ func (delta *Delta) Wait() error {
 	return <-delta.done
 }
 
+func checksumsMatch(a, b contracts.Entity) bool {
```

Comment:
I would move this function after `Delta.run()` so that private functions come after public ones and `Delta` methods stay together.